cookieDriver.js

```diff
@@ -0,0 +1,290 @@
+/*!
+ * Copyright (c) 2018 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+import {get, getAll, remove, set} from 'tiny-cookie';
+
+export default {
+    _driver: 'cookieWrapper',
+    _initStorage,
+    iterate,
+    getItem,
+    setItem,
+    removeItem,
+    clear,
+    length,
+    key,
+    keys,
+    dropInstance
+};
+
+function _initStorage(options) {
+  const dbInfo = {};
+  if(options) {
+    Object.assign(dbInfo, options);
+  }
+  dbInfo.keyPrefix = _getKeyPrefix(options, this._defaultConfig);
+  // FIXME: support `serializer`?
+  //dbInfo.serializer = serializer;
+  this._dbInfo = dbInfo;
+  return Promise.resolve();
+}
+
+async function clear(callback) {
+  try {
+    await this.ready();
+    _clear(this._dbInfo.keyPrefix);
+  } catch(e) {
+    if(callback) {
+      return callback(e);
+    }
+    throw e;
+  }
+  if(callback) {
+    callback();
```

Comment:
add return for safety.

Reply:
The contract for this API (which is up to `localForage`) seems to be to return undefined, not a function.

Reply:
does this mean that this would work?
```
callback();
return;
```

Reply:
Yeah, but nothing happens after that ... so that leads me to think it would just be confusing/unnecessary.